### DISCUSSION TITLE
Welcome to SANSA-Stack Discussions!

### DISCUSSION THREAD

**Discussion by JensLehmann** [SANSA-Stack/SANSA-RDF#132] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by carstendraschner** [comment]
Hello Everybody,
I'm Carsten a PhD candidate at SDA research group. I plan to develop modules for the SANSA-ML Stack to make the interaction with large scale RDF Data convenient for Data Scientists and Machine Learning Engineers as they might be used to from common python libraries like sci-kit learn and Apache Spark MLlib. I would be curious to know who else is developing those approaches and might be interested to contribute there as well.
